src/main/java/cms/gov/madie/measure/resources/BundleController.java

```diff
@@ -0,0 +1,43 @@
+package cms.gov.madie.measure.resources;
+
+import cms.gov.madie.measure.exceptions.ResourceNotFoundException;
+import cms.gov.madie.measure.repositories.MeasureRepository;
+import cms.gov.madie.measure.services.BundleService;
+import cms.gov.madie.measure.utils.ControllerUtil;
+import gov.cms.madie.models.measure.Measure;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+import java.util.Optional;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class BundleController {
+
+  private final MeasureRepository measureRepository;
+
+  private final BundleService bundleService;
+
+  @GetMapping(path = "/measures/{measureId}/bundles", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
How about using the singular, `bundle`, for the end point to indicate the bundling action?

Reply:
Isn't plural nouns recommended for resources?

Reply:
`/bundles` sounded off to me since we cannot access `/bundle/{id}`. Of course, that could just be my personal opinion not backed by anything.

So I went back to this [REST naming guide](https://restfulapi.net/resource-naming/). Reading through the different Resource types, it sounded to me that either Controller Resource (2.1.4) or document resource archetype (end of 2.1.1) match up best.  Take a look and let me know what you think.